tools/tm-bench/main.go

```diff
@@ -68,6 +68,14 @@ Examples:
 		fmt.Printf("Running %ds test @ %s\n", durationInt, flagSet.Arg(0))
 	}
 
+	if txSize < 40 {
+		fmt.Fprintln(
+			os.Stderr,
+			"txSize must be greater than or equal to 40.",
```

Comment:
`The size of a transaction must be ...`

Users are not aware of txSize (only tm-bench developers know what txSize is!)